gradle.properties

```diff
@@ -19,52 +19,16 @@ org.gradle.vfs.watch=true
 org.gradle.jvmargs=-Xmx6g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 
 # kotlin
-kotlin_version=1.6.20
 kotlin.native.ignoreDisabledTargets=true
 kotlin.mpp.stability.nowarn=true
 kotlin.mpp.enableCompatibilityMetadataVariant=true
+kotlin.js.compiler=both
+kotlin.mpp.enableCInteropCommonization=true
+kotlin.internal.mpp.hierarchicalStructureByDefault=true
 
-# kotlin libraries
+kotlin_version=1.6.20
 coroutines_version=1.6.0
 atomicfu_version=0.17.0
-validator_version=0.8.0
-kotlinx_html_version=0.7.5
-
-# server
-netty_version=4.1.72.Final
-netty_tcnative_version=2.0.44.Final
-jetty_version=9.4.44.v20210927
-jetty_alpn_api_version=1.1.3.v20160715
-json_simple_version=1.1.1
-java_jwt_version=3.13.0
-jwks_rsa_version=0.17.0
-tomcat_version=9.0.56
-
-# utility
-logback_version=1.2.10
-junit_version=4.13.2
-dokka_version=1.4.32
-nodejs_version=14.17.1
-npm_version=6.4.1
 slf4j_version=1.7.36
```

Comment:
if those are used inside `buildSrc` or just in build script, they can be replaced with something like this, to avoid storing versions in `gradle.properties`:
```kotlin
val versionCatalog = extensions.getByType<VersionCatalogsExtension>().named("libs")
api(versionCatalog.findLibrary("slf4j-api").get()) //or implementation
```

also, it's possible to get version in the same way, look at: https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/VersionCatalog.html